src/app/page.tsx

```
@@ -1,59 +1,37 @@
-  const gridProjects = Array.from({ length: 17 }, (_, i) => ({
-    id: i + 4,
-    rank: i + 4,
-    image: "/project-test.png",
-    sellerName: "판매자 이름",
-    projectName: "상품 이름",
-    achievementRate: 0.8,
-    size: 160,
-    daysLeft: 10,
-    liked: false,
-  }))
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: loadRankProjects

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)